src/cascadia/ut_app/JsonTests.cpp

```diff
@@ -94,10 +94,10 @@ namespace TerminalAppUnitTests
         const auto schemeObject = VerifyParseSucceeded(campbellScheme);
         auto scheme = implementation::ColorScheme::FromJson(schemeObject);
         VERIFY_ARE_EQUAL(L"Campbell", scheme->Name());
-        VERIFY_ARE_EQUAL(til::color{ARGB(0, 0xf2, 0xf2, 0xf2)}, scheme->Foreground());
-        VERIFY_ARE_EQUAL(til::color{ARGB(0, 0x0c, 0x0c, 0x0c)}, scheme->Background());
-        VERIFY_ARE_EQUAL(til::color{ARGB(0, 0x13, 0x13, 0x13)}, scheme->SelectionBackground());
-        VERIFY_ARE_EQUAL(til::color{ARGB(0, 0xFF, 0xFF, 0xFF)}, scheme->CursorColor());
+        VERIFY_ARE_EQUAL(til::color{ ARGB(0, 0xf2, 0xf2, 0xf2) }, scheme->Foreground());
+        VERIFY_ARE_EQUAL(til::color{ ARGB(0, 0x0c, 0x0c, 0x0c) }, scheme->Background());
+        VERIFY_ARE_EQUAL(til::color{ ARGB(0, 0x13, 0x13, 0x13) }, scheme->SelectionBackground());
+        VERIFY_ARE_EQUAL(til::color{ ARGB(0, 0xFF, 0xFF, 0xFF) }, scheme->CursorColor());

```

Comment:
til::color already takes argb as its normal constructor params, is the bit where we go from A R G B -> COLORREF necessary?